py3status/modules/sysdata.py

```diff
@@ -60,18 +60,14 @@
 
 import re
 
-ONE_KIB = pow(1024, 1)  # 1 KiB in B
-ONE_MIB = pow(1024, 2)  # 1 MiB in B
-ONE_GIB = pow(1024, 3)  # 1 GiB in B
```

Comment:
We don't need `from __future__ import division` too?

Reply:
Not sure. There are some divisions left, e.g., `used_mem_p = 100 * used_mem_kib / total_mem_kib`

Reply:
`memi[tok[0]] = float(tok[1])` gives us float values so we definitely can remove it. You can test this with `self.py3.log()`, eg `self.py3.log('=== Float? {} and {}'.format(used_mem_kib, total_mem_kib))` and run `python2 sysdata.py` in terminal.